cfgov/v1/models/molecules.py

```diff
@@ -75,3 +77,41 @@ class TextIntroduction(blocks.StructBlock):
     class Meta:
         icon = 'title'
         template = 'v1/demo/molecules/text_introduction.html'
+
+
+class ContactAddress(blocks.StructBlock):
+    label = blocks.CharBlock(max_length=50)
+    title = blocks.CharBlock(max_length=100, required=False)
+    street = blocks.CharBlock(max_length=100)
+    city = blocks.CharBlock(max_length=50)
+    state = blocks.CharBlock(max_length=25)
+    zip_code = blocks.CharBlock(max_length=15, required=False)
+
+    class Meta:
+        template = 'wagtail/molecules/contact-address.html'
+        icon = 'mail'
+        label = 'Address'
+
+
+class ContactEmail(blocks.StructBlock):
+    emails = blocks.ListBlock(atoms.Hyperlink(label='Email'))
+
+    class Meta:
+        icon = 'mail'
+        template = 'wagtail/molecules/contact-email.html'
+        label = 'Email'
+
+
+class ContactPhone(blocks.StructBlock):
+    fax = blocks.BooleanBlock(default=False, required=False, label='Fax?')
+    phones = blocks.ListBlock(
+        blocks.StructBlock([
+            ('number', blocks.CharBlock(max_length=15)),
+            ('vanity', blocks.CharBlock(max_length=15, required=False)),
+            ('tty', blocks.CharBlock(max_length=15, required=False)),
+        ]))
+
+    class Meta:
+        icon = 'mail'
```

Comment:
This looks like it's phone contact method, is `mail` the correct icon choice here?